src/main/java/com/iota/iri/service/API.java

```diff
@@ -1069,6 +1170,21 @@ private void sendResponse(final HttpServerExchange exchange, final AbstractRespo
         sinkChannel.resumeWrites();
     }
 
+    private String convertResponseToClientFormat(AbstractResponse res) {
+        String response = null;
+        if(res instanceof IXIResponse){
+            final String content = ((IXIResponse)res).getContent();
+            if(content != null && !content.equals("")){
```

Comment:
Should `"   "` pass?

Use `StringUtils.isNotEmpty()` or `StringUtils.isNotBlank()` according to your answer